src/com/phonebookservice/util/ErrorCode.java

```diff
@@ -0,0 +1,14 @@
+package com.phonebookservice.util;
+
+/**
+ * Error messages.
+ *
+ * @author Marwa Saleh
+ */
+public abstract class ErrorCode {
```

Comment:
Why Error code is class ? it should be enum, right ?

Reply:
we discussed that to leave it as a class cause we will need it in other services